build.cmd

```diff
@@ -393,14 +349,10 @@ if "%InstallPythonPackages%" == "True" (
     echo "Installing python packages ... "
     echo "#################################"
     call "%PythonExe%" -m pip install --upgrade "pip==19.3.1"
```

Comment:
I remember that there were a few issues with newer versions of `pip`. Are these issues still there or can the version of `pip` be upgraded to the latest?

Reply:
not relevant to this PR. will address it separately

---
In reply to: [441272014](https://github.com/microsoft/NimbusML/pull/494#discussion_r441272014) [](ancestors = 441272014)